src/printer/index.ts

```diff
@@ -103,8 +103,8 @@ export function print(path: AstPath, opts: ParserOptions, print: printFn): Doc {
 			const isSelfClosingTag =
 				isEmpty && (node.type !== 'element' || selfClosingTags.includes(node.name));
 
-			const attributeLine =
-				opts.singleAttributePerLine && node.attributes.length > 1 ? breakParent : '';
+			const isSingleLinePerAttribute = opts.singleAttributePerLine && node.attributes.length > 1;
+			const attributeLine = isSingleLinePerAttribute ? breakParent : '';
```

Comment:
seems like a weird variable name for a join deliminator for the attribute line?

Reply:
This entire part might need a slight refactor eventually because it's a bit confusing to go through 😅